### PR TITLE
HDFS-9797. Log Standby exceptions thrown by RequestHedgingProxyProvid…

### PR DESCRIPTION
**Pull request by benallamar**
…er at DEBUG Level (Inigo Goiri via asuresh)

### CODE REVIEW

**Comment by ashangit** [comment]
LGTM

File: hadoop-common-project/hadoop-common/CHANGES.txt [hunk 1]
```diff
@@ -4,6 +4,1505 @@ Criteo Release
...
+    HADOOP-12691. Add CSRF Filter for REST APIs to Hadoop Common.
+    (Larry McCay via cnauroth)
+
+    HADOOP-12635. Adding Append API support for WASB. (Dushyanth via cnauroth)
+
+    HADOOP-12426. Add Entry point for Kerberos health check
+    (Steve Loughran via cnauroth)
+
+    HADOOP-12548. Read s3a creds from a Credential Provider.
+    (Larry McCay via cnauroth)
```
Comment (ashangit) [inline hadoop-common-project/hadoop-common/CHANGES.txt]: Please only set the patch you add

**Comment by ashangit** [comment]
It seems that patch HDFS-9797 has been merged with HDFS-11395, could you please split them

**Comment by benallamar** [comment]
I've taken your patch @ashangit and I did fix the line that seems weired to me. I tested on the EXP and it seems to work fine

**Pull request merged**